spec/usermanual/teachers.html.erb

```diff
@@ -93,6 +93,7 @@ require 'cgi' # for escapeHTML
     <ul>
       <li><b>Name</b> of the organization that will be seen in the app</li>
       <li><b>Description</b> about your organization, seen in this organization's page</li>
+      <li><b>Logo</b> for your organization, seen in this organization's page (you can change this and other fields above later)</li>
```

Comment:
Was there some requirements for the logo, like minimum size? If not, good, if there is, would be good to tell them here.